prerequisites.rst

```diff
@@ -1,35 +1,280 @@
+named Debian GNU/Linux as "the universal operating system" [9]_ 
+
+**Supported Debian OS versions can be checked in the** :doc:`releases section </releases>`.
+
+While possible to deploy in a virtual environment, depending of the nature of
+virtualization it will degrade the performance access in various ways. Installation
+in LXC or any other container based solution is not supported.
+
+Whatever the situation, the |omv| assumes that target operating system does not
+have any previous installation of any of the programs used by |omv| dependencies:
+
+=============  ==============  =================  ==========================================
+ software       package          related to        Observations
+=============  ==============  =================  ==========================================
+ http server    nginx           |webui|            port 80 must be free at installation
+ http system    php             |webui|            only OS php packages wil be used!
+ display GUI    lighdm,xdm..    Desktop install    **Conflicts, any will be deinstalled**
+ network man    netplan.io      Networking         network is managed by |omv| with netplan
+ ssh server     ssh             Remote access      any configuration will be managed
+ smb server     samba           Shared drives      any configuration will be managed
+ quota man      quota           Quota manage       any configuration will be managed
+=============  ==============  =================  ==========================================
+
+The device boot (BOOT)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The boot type and its support is determined by the Debian operating system
+which supports BIOS, UEFI, PXE, UBOOT and many others, the configurations
+of these depend on the installation of the operating system and once done
+the |omv| can be installed without problems.
+
+However the downloadable and ready-to-use ISO image from the official |omv| website
+only supports AMD64 with BIOS mode boot at the moment. If you want to have |omv| on
+other computers with another boot type/mode you should install Debian first
+and then manually the |omv| can be installed without problems.
+
+**Supported Debian OS versions can be checked in the** :doc:`releases section </releases>`.
+
+The system drive storage (SDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any brand of storage disk will be supported as long as it is supported by
+the Debian operating system version, since it is where |omv| system resides
+and runs, which is Debian GNU/Linux, check it at the :doc:`releases section </releases>`.
+
+The system storage disk will be fully managed and taken over by the |omv| after
+installed. In case of the official ISO installation, will automatically partition
+the system storage disk drive into 3 partitons. Consult the next table for.
+
+In case of a manual installation on a previous Debian operating system, this
+**drive must have at least two partitions**. Consult the next table for.
```

Comment:
Users can install the OS on one partition, it is NOT a MUST; but it is suggested to use several partitions.
```suggestion
**drive should have at least two partitions**. Consult the next table for.
```

Reply:
commit misspelled with 70f443af661407cfca3c794da2723e26d987bf2e